doc-content/enterprise-only/project-data/test-scenario-editor-con.adoc

```diff
@@ -0,0 +1,4 @@
+[id='test-scenario-editor-con']
+= Using the Test Scenario editor in {CENTRAL}
+
+In {PRODUCT}, there are two test scenarios editors. One is the *Test Scenario* editor and the other is *Test Scenario (Preview)* editor. The *Test Scenario (Preview)* editor is a technology preview only and is not supported for production use. Though, you can use either of them to test your business rules & data. This chapters covers how to use the *Test Scenario* editor to create and run test scenarios.
```

Comment:
"Though, you can use ..."
Suggestion: "However, you can use ..."

Reply:
Changed as suggested.